Try fix behaviour when network is not available at bootup

--- ./ntp_sources.c.orig
+++ ./ntp_sources.c
@@ -76,7 +76,7 @@
 
 #define RESOLVE_INTERVAL_UNIT 7
 #define MIN_RESOLVE_INTERVAL 2
-#define MAX_RESOLVE_INTERVAL 9
+#define MAX_RESOLVE_INTERVAL 6
 
 static struct UnresolvedSource *unresolved_sources = NULL;
 static int resolving_interval = 0;
